org.telegram.desktop.json

```diff
@@ -38,6 +38,14 @@
     "cleanup-commands": [
         "mkdir -p ${FLATPAK_DEST}/lib/ffmpeg"
     ],
+    "build-options": {
+        "cflags": "",
```

Comment:
Why not `-O2 -g`?

Reply:
cmake sets all base flags on its own

Reply:
Ok, we should set `-DCMAKE_BUILD_TYPE=RelWithDebInfo` for mozjpeg, then.

Reply:
Yeah, indeed

Reply:
And `-DCMAKE_BUILD_TYPE=Release` for yasm to avoid creating useless debug info